index.html

```diff
@@ -152,91 +174,141 @@ <h2 class="work-title">Multi-Post Stories</h2>
-                        </li>
-                    </ul>
-                </li>
-                <ul class="about-ul">
-                    <h3 class="sub-title">Frameworks <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down"></h3>
-                </ul>
-                <ul class="about-ul">
-                    <h3 class="sub-title">Skills <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down"></h3>
-                </ul>
-            </ul>
-        </section>
-        
-        <section class="contact--section">
-          <div>
-            <form action="https://formspree.io/f/xwkjylde" method="post" class="form">
-                <h2 class="contact-me">Contact me</h2>
-                <p class="contact-text">
-                    If you have an application you are interested in developing, 
-                    a feature that you need built or a project that needs coding. 
-                    I'd love to help with it!
-                </p>
-              <input type="text" name="username" 
-              class="user" required placeholder="Write your name here" maxlength="30">
-              <input type="email" name="useremail" class="user" required placeholder="Email address">
-              <textarea name="text" class="textarea" maxlength="500" minlength="1" 
-              cols="30" rows="10" required placeholder="Write your message here ..."> </textarea>
-              <input type="submit" name="submit" class="button" value="Get in touch">
-            </form>
-          </div>
-        </section>
+          </section>
+      <section class="contact_form" id="contact">
+        <div class="contact_container">
+          <h2 class="title">Contact me</h2>
+          <p class="contact_form_des">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I&#39;d
+            love to help with it
+          </p>
+          <form
+            class="form_container"
+            id="contact-form"
+            method="POST"
+            action="https://formspree.io/f/xeqwyqrv"
+          >
+            <fieldset class="form_inputs">
+              <input
+                class="form_input"
+                name="name"
+                id="name"
+                type="text"
+                placeholder="Your Name"
+                maxlength="30"
+                required
+                />
+              <input
+                class="form_input"
+                name="email"
+                id="email"
+                type="email"
+                placeholder="example@me.com"
+                required
+                />
+              <textarea
+                class="form_text_area"
+                name="message"
+                id="message"
+                placeholder="Write your message here"
+                maxlength="500"
+                required>
+            </textarea>
```

Comment:
- [ ] You did a great job setting up HTML validations and integrating the Formspree service. :+1:  Let's work on some minor details to achieve the project requirements.
  - Currently, the placeholder of your `textarea` is not getting displayed because you have whitespaces between the opening and the closing tags of your `textarea`. Kindly fix it by removing the whitespace. 😎️
  - Please decrease the spacing to the left and the right of the input fields as it seems to bit a bit bigger when compared to the design. 🤝️
  - Kindly align the submit button to the left side as displayed on the design guideline. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/Uzair-Manzoor/Portfolio/assets/17243325/a9bc0a46-5f60-4d13-a192-027211b94f8c" width="300">|<img src="https://github.com/Uzair-Manzoor/Portfolio/assets/17243325/7cb3742d-5c1b-4803-82b4-45326c32b8ae" width="300">|